### ISSUE TITLE
[JAWS + IE11] Cannot read the entire About dialogue

### ISSUE THREAD

**Issue by phet-steele** [phetsims/john-travoltage#243]
@jessegreenberg, when entering the About... dialog, JAWS begins reading all the content until "Carl Wieman" in the credits. From there, JAWS randomly says "dialogue" and begins reading all the content over from the beginning. The second time JAWS reaches "Carl Wieman", it skips the remaining content and goes straight to the link you are focusing. So it seems you cannot hear all the content in this dialogue since something near "Carl Wieman" causes JAWS to stop.

For phetsims/tasks#831.

**Comment by jessegreenberg** [comment]
Thanks @phet-steele, the first think I want to check is if there is a setting for reading length.  The element with this content looks like
```html
<p tabindex="-1" id="75-175-245-244-243-238-220">‪‪‪‪‪‪‪‪‪‪‪‪‪‪‪Lead Design: ‪Noah Podolefsky, Carl Wieman, Sam Reid‬‬‬‬‬‬‬‬‬‬‬‬‬‬‬‬</p>
```

Nothing about this makes me think that JAWS should stop after "Wieman".

**Comment by jessegreenberg** [comment]
For me, JAWS reads up intil the "Credits" heading and then nothing more.

**Comment by jessegreenberg** [comment]
Not likely a JAWS setting, this is not a problem for JAWS + Chrome.

**Comment by jessegreenberg** [comment]
It is not a problem in this example:
https://jsfiddle.net/v4xa9x6k/2/show/

**Comment by jessegreenberg** [comment]
It is a problem in this example case as well for IE11 + JAWS.
https://jsfiddle.net/v4xa9x6k/2/show/

**Comment by jessegreenberg** [comment]
In that example it doesn't even read "Credits", it gets interrupted with the Dialog role, it says "CreDialog, John Travoltage..." and then the rest until Credits, and then it skips to the link.

**Comment by jessegreenberg** [comment]
When I change the role on the dialog from "dialog" to "alertdialog", the problem goes away in the JSFiddle example.

**Comment by jessegreenberg** [comment]
That seemed to be a fluke, it still has this problem.

**Comment by jessegreenberg** [comment]
I tried adding role=document on the dialog content under the element with role=dialog, it has no effect.

**Comment by jessegreenberg** [comment]
Here is an example with the HTML semantics inside of the Dialog removed:
https://jsfiddle.net/v4xa9x6k/5/show/

Cursor got to the "S" in "Sam Reid", and then went back to the top of the dialog.

**Comment by jessegreenberg** [comment]
There are two issues here.

1) Reading stops at about "Credits"
2) Once it hits that location it repeats dialog content again until it reaches "Credits" again, then it jumps to the focused link.

The repetition is happening because there isn't a proper label associated with the dialog.  Fixing the aria-labelledby association will fix this.

I am not sure why it would stop reading at credits.  It might be a JAWS feature that reduces verbocity.

**Comment by jessegreenberg** [comment]
The above commit fixes the label association and JAWS doesn't read through the dialog twice anymore.  Still not sure why it would stop at "Credits".

**Comment by jessegreenberg** [comment]
@jobara recommended that we test against this example: http://jqueryui.com/dialog/#default

In a test case JAWS read through everything.  When I make the content longer (https://jsfiddle.net/9nkq8sL6/4/) JAWS gets to the word "Resized" in the second paragraph, then stops, so it seems that JAWS stops reading after a certain length.

**Comment by jessegreenberg** [comment]
@jobara noticed that the JSFiddle examples above had a missing closing tag. 
 https://jsfiddle.net/9nkq8sL6/6/embedded/result/ has this fixed.  The problem is still there after the update.  Behavior is different depending on context.

In https://jsfiddle.net/9nkq8sL6/6/embedded/result/, once focus lands on the close button, JAWS will read until near the bottom of the second paragraph and then loop back to the top and read everything.

**Comment by jessegreenberg** [comment]
In this case: https://jsfiddle.net/9nkq8sL6/6/ (Same HTML, but rendered at a different time)
JAWS reads until "resized" in the paragraph and then stops reading.

**Comment by jessegreenberg** [comment]
@jobara recommended that we take a look at the accessibility tree using aViewr from the Paciello group to see the information that is provided to the AT.

**Comment by jessegreenberg** [comment]
I am not able to get aViewer to work on my Windows 10 Machine, it is crashing consistently when I try to view the content.

**Comment by jessegreenberg** [comment]
@emily-phet do you think that this is worth more exploration?  The content can still be easily read by the virtual cursor, it is just that the initial automatic read through the About Dialog stops at the "Credits" section.

**Comment by emily-phet** [comment]
@jessegreenberg I think we can hold off on this for now, considering the content is still accessible using shortcuts, and the content is required for useful sim exploration. We should keep in mind for future reference that there appears to be a word limit for JAWS.

**Comment by jessegreenberg** [comment]
Sounds good @emily-phet.  I think we are guessing that this is an issue with IE11.  Would you like us to submit a bug report, or just close this issue?

**Comment by jessegreenberg** [comment]
I made the above issue where we decide whether or not to submit a bug report.  Documentation has been moved to that issue. Closing.
